addons/sms/wizard/sms_composer_views.xml

```diff
@@ -47,9 +47,8 @@
                             <field name="recipient_single_description" class="oe_inline"/>
                             <field name="recipient_single_number_itf" class="oe_inline" nolabel="1" options="{'onchange_on_keydown': True}" placeholder="e.g. +1 415 555 0100"/>
                         </div>
-                                                
-                        <field name="body" widget="sms_widget" attrs="{'invisible': ['|', ('comment_single_recipient', '=', False), ('recipient_single_valid', '=', True)]}"/>
-                        <field name="body" widget="sms_widget" attrs="{'invisible': [('comment_single_recipient', '=', True), ('recipient_single_valid', '=', False)]}" default_focus="1"/>
+                        <field name="body" widget="sms_widget" attrs="{'invisible': ['|', ('comment_single_recipient', '=', False), ('recipient_single_valid', '=', True)], 'required': ['|', ('comment_single_recipient', '=', False), ('recipient_single_valid', '=', True)]}"/>
+                        <field name="body" widget="sms_widget" attrs="{'invisible': [('comment_single_recipient', '=', True), ('recipient_single_valid', '=', False)], 'required': [('comment_single_recipient', '=', True), ('recipient_single_valid', '=', False)]}" default_focus="1"/>
```

Comment:
Those changes can be undone as the field is required again

Reply:
I was about to remove it during the last push but saw that it avoids displaying the same field twice in the invalid fields error so the required=True in the model had no effect on the desired behavior specified in the task :sweat_smile: 

![image](https://user-images.githubusercontent.com/88150098/156382469-4017275b-61dd-459d-ad7a-137ef5effb30.png)

Reply:
Ok ok makes sense now